src/App.tsx

```diff
@@ -1,61 +1,142 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+
+function findUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function findUserByName(userName: string) {
+  return usersFromServer.find(user => user.name === userName) || null;
+}
+
+const preparedTodoList: Todo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: findUserById(todo.userId),
+  };
+});
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState('');
+
+  const [todoList, setTodoList] = useState(preparedTodoList);
+
+  const [isErrorOnUserSelect, setErrorOnUserSelect] = useState(false);
+  const [isErrorOnTitleInput, setErrorOnTitleInput] = useState(false);
+
+  const shouldResetForm = () => {
```

Comment:
Looks like boolean variable but this is function

Reply:
done